src/test/test-path-util.c

```diff
@@ -69,15 +78,28 @@ static void test_path(void) {
         assert_se(fd >= 0);
         assert_se(fd_is_mount_point(fd, "/", 0) > 0);
 
-        {
-                char p1[] = "aaa/bbb////ccc";
-                char p2[] = "//aaa/.////ccc";
-                char p3[] = "/./";
-
-                assert_se(path_equal(path_kill_slashes(p1), "aaa/bbb/ccc"));
-                assert_se(path_equal(path_kill_slashes(p2), "/aaa/./ccc"));
-                assert_se(path_equal(path_kill_slashes(p3), "/./"));
-        }
+        test_path_kill_slashes_one("aaa/bbb////ccc", "aaa/bbb/ccc", "aaa/bbb/ccc");
+        test_path_kill_slashes_one("//aaa/.////ccc", "/aaa/./ccc", "/aaa/ccc");
+        test_path_kill_slashes_one("///", "/", "/");
+        test_path_kill_slashes_one("///.//", "/.", "/");
```

Comment:
Hmm, this removes a trailing slash, and a trailing slash is significant in some contexts. I'm not sure if this matters here.

Reply:
trailing slash? I think I do not change anything about slashes.